src/Ydb.Sdk/src/Services/Sessions/SessionBase.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.Extensions.Logging;
+using Ydb.Sdk.Client;
+
+namespace Ydb.Sdk.Services.Sessions;
+
+public abstract class SessionBase : ClientBase, IDisposable
+{
+    internal static readonly TimeSpan DeleteSessionTimeout = TimeSpan.FromSeconds(1);
+
+
+    public string Id { get; }
+    internal string? Endpoint { get; }
+
+    private protected bool Disposed;
+    protected readonly ILogger Logger;
+
+
+    protected SessionBase(Driver driver, string id, string? endpoint, ILogger logger) : base(driver)
```

Comment:
Is is possible to create own session outside sdk?
If not - what about protected internal for the constructor?